### ISSUE TITLE
Clean up master branch

### ISSUE THREAD

**Issue by Gregory94** [Gregory94/LaanLab-SATAY-DataAnalysis#44]
The [master branch](https://github.com/Gregory94/LaanLab-SATAY-DataAnalysis) had a major overhaul. It is merged with was previously the [satay_processing branch](https://github.com/Gregory94/LaanLab-SATAY-DataAnalysis/tree/satay_processing). This branch won't be updated anymore. The old master branch is now called [developercodes](https://github.com/Gregory94/LaanLab-SATAY-DataAnalysis/tree/developercodes).

**Comment by leilaicruz** [comment]
Indeed, the master branch should be the working branch of the repository, so **no pushes should be made to this branch that concerns the main working code** . If you would like to modify a function to your own needs and you require to push  your changes to the remote repository (because you want to show it to someone) create a new branch for that.
